packit.spec

```diff
@@ -57,49 +30,48 @@ Requires:       rpm-build
 Requires:       rpmdevtools
 # Copying files between repositories
 Requires:       rsync
-%if 0%{?rhel}
-# rhbz#1968618 still not fixed for epel-8
-Requires:       python3-koji
-%endif
-%{?python_provide:%python_provide python3-%{real_name}}
 
-%description -n python3-%{real_name}
+%description -n python3-%{name}
 Python library for Packit,
 check out packit package for the executable.
 
 
 %prep
 %autosetup -n %{pypi_name}-%{version}
-# Remove bundled egg-info
-rm -rf %{pypi_name}.egg-info
 
-%if 0%{?rhel}
-# rhbz#1968618 still not fixed for epel-8
-sed -i -e 's|koji|# koji|' setup.cfg
+
+%generate_buildrequires
+%if 0%{?rhel} && 0%{?rhel} <= 9
+# Testing packages missing on epel9
+%pyproject_buildrequires -w
+%else
+%pyproject_buildrequires -x testing
 %endif
 
+
 %build
-%py3_build
+%pyproject_wheel
+
 
 %install
-%py3_install
-python3 setup.py --command-packages=click_man.commands man_pages --target %{buildroot}%{_mandir}/man1
+%pyproject_install
+%pyproject_save_files %{name}
+PYTHONPATH="%{buildroot}%{python3_sitelib}" click-man %{name} --target %{buildroot}%{_mandir}/man1
 
 install -d -m 755 %{buildroot}%{_datadir}/bash-completion/completions
-cp files/bash-completion/packit %{buildroot}%{_datadir}/bash-completion/completions/packit
+cp files/bash-completion/%{name} %{buildroot}%{_datadir}/bash-completion/completions/%{name}
```

Comment:
Use `%{bash_completions_dir}` instead of `%{datadir}/bash-completion/completions`

Reply:
This wasn't fixed, so I'm not sure why it was marked resolved.

Reply:
Weird, this one is fixed, but I noticed there is another spot a few lines below. It must not be picked up because of rebasing